SomosMasApp/SomosMasApp/App/Shared/Modules/Home/ViewModel/HomeViewModel.swift

```diff
@@ -78,5 +79,52 @@ class SliderViewModel {
         return newsResponse.count
     }
 
+    func allServicesLoading() {
+        let dispatchGroup = DispatchGroup()
+
+        func loadSliders() {
+             dispatchGroup.enter()
+             getSliders()
+             dispatchGroup.leave()
```

Comment:
the get sliders is async so the .leave here will happen inmediatly you should leave the group after we get the response as we talk about, same with the other services